assignment-2/7s.md

```diff
@@ -1,2 +1,296 @@
+##### Strategy
+KLM's main strategy is to position itself as a luxury brand. In their advertisement they position flying as something
+unique and beautiful, instead of focusing on the price of the tickets. Instead of nickle-and-diming their customers for drinks and food like many other airlines, they prefer to just include it in the ticket price. This gives a more luxurious and relaxed feeling during flights. Together with this they also aim for a very user friendly
+atmosphere by focusing a great deal of attention to twitter and facebook customer service.
+
+In order to establish a better market position, KLM is a member of an airline alliance: Skyteam. Through these alliances airlines expand their global network of destinations. Through this network the airlines can offer a better service to their customers, they can integrate the flights of their allied airlines (this is called codesharing). This helps customers in making it easier to book multi-legged trips, thereby creating a one-stop-shop for virtually any destination in the world. Furthermore, it is customary for alliances to share frequent flyer miles. Which makes customers more likely to choose for an ally of their regular airline when needing to select a different airline, as such creating a stronger binding to their customer.
+It has to be noted that this is not a very uncommon strategy. According to the Skytrax (http://www.airlinequality.com/) rating (which is most likely not a very scientific measurement, but it is still a recognized industry benchmark) there is only one five-star airline not participating in an alliance: Hainan Airlines. Likewise, a majority (21 of 38) of the airlines that are rated with four stars are participating in an ailliance.
+##### Structure
+As to be expected of a company with 32.000 employees, the structure is very classical and hierarchical.
+The executive committe consists out of 13 people [5], each responsible for their subdivison of KLM. This kind of structure is not the best for quick decision making, but it is good at managing very large companies. The main division is done by function. Every occupation is in their own subdivison. Within these divisions there can be further subdivisions as well, for example the pilots have subdivisions based on the type of aircraft that they are flying. For such a global company you would expect that there is a more geographical structure. But pretty much every flight is either from or to Amsterdam or a layover of one of those flights. This means that most of the employees are living in the Netherlands. Of course there are always some exceptions, for example KLM has Asia based flight attendants to help reduce the language barrier on all of their flights to and from asian destinations.
```

Comment:
"The executive committe consists out of 13 people" => The use of out is kind of unnecessary here.